app/views/donations/show.html.erb

```diff
@@ -49,7 +49,28 @@
       </div>
     </div>
   </div>
-
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-12">
+        <div class="card">
+          <div class="card-header">
+            <h2 class="card-title">Notes</h2>
+          </div>
+          <div class="card-body p-0">
+              <div class="card">
+                <div class="col-xs-12 col-sm-12" id="donation-notes">
+                  <% if @donation.comment %>
+                    <p><%= simple_format(@donation.comment) %></p>
+                  <% else %>
+                    <p> None provided </p>
```

Comment:
```suggestion
                    <p class='font-italic'> None provided </p>
```
Let's make this italics to make some small visual distinct between an actual comment or not. Source - https://getbootstrap.com/docs/4.0/utilities/text/#font-weight-and-italics